RELEASE.md

```diff
@@ -0,0 +1,75 @@
+# Release management for protocol buffer specifications
+
+This repository primary provides two features:
+
+* Protobuf specifications for messages used within Sigstore.
+* Language clients for different ecosystems.
+
+During a release, a few steps have to be synchronized to release the
+messages and the language clients.
+
+## Notes on semantic versioning
+
+### Major version change
+As expected this indicate a breaking change. Any major update MUST
+update the package name of the generated code.
+
+### Minor version change
+An update which does not break the functionality of existing (older)
+clients. For more information on forward compatible changes in
+protobuf see the [Language
+Guide](https://developers.google.com/protocol-buffers/docs/proto3#updating).
+
+### Patch version change
+Any update which does not change the behaviour. For the protocol buffer
+messages this is limited to _only_ capture changes in the comments,
+not the messages themselves. For language clients patch versions MAY
+be used for bug-fixes.
+
+## Releasing new versions of the messages
+
+Checklist prior to releasing:
+
+1. Gather consensus among the community and maintainers of this
```

Comment:
This might be pedantic, but do you want to define consensus? all or majority of maintainers? define it as "inform the community of its release, provide a comment period of X days, then release?"

Reply:
Agree, something that's easier to quantize is better.